README.md

```diff
@@ -65,6 +65,12 @@ config :geocoder, Geocoder.Worker, [
   httpoison_options: [proxy: "my.proxy.server:3128", proxy_auth: {"username", "password"}]
 ]
 ```
+
+If you want to change the cache precision (defaults to `6`):
```

Comment:
Would be good to describe here _why_ you'd want to increase the precision (maybe even link to the related issue). It's a non-obvious problem for those new to the package.

Going to merge this in without the change, feel free to submit a new PR with this change if you'd like!